web/themes/custom/digital_gov/templates/node/node--community--teaser.html.twig

```diff
@@ -1,130 +1,16 @@
- * - page: Flag for the full page state. Will be true if view_mode is 'full'.
- *
- * @see template_preprocess_node()
- *
- * @ingroup themeable
+ * Theme implementation to display a Commuinity node as a teaser.
  */
 #}
-{% set title = node.label %}
-{% set summary = content.field_summary|field_value %}
-{% set member_count = content.field_members|field_value %}
-{% set logo_url %}{% apply spaceless %}
-  {% if content.field_dg_logo|field_value %}
-    {{ node.field_dg_logo|file_uri }}
-  {% endif %}
-{% endapply %}{% endset %}
-{% set dg_acronym = content.field_dg_acronym|field_value %}
-{% set url = url('entity.node.canonical', {node: node.id}) %}
-{% set short_name = content.field_dg_shortname|field_value %}
-<div class="usa-card cop-card--{{ dg_acronym }}">
-  <div class="usa-card__container">
-    {% if title %}
-      <div class="usa-card__header">
-        <h2 class="usa-card__heading">{{ title }}</h2>
-      </div>
-    {% endif %}
-    {% if logo_url %}
-      <div class="usa-card__media usa-card__media--indent">
-        <div class="usa-card__img">
-          {{ drupal_image(logo_url, '', {alt: short_name|render ~ ' logo'}) }}
-        </div>
-      </div>
-    {% endif %}
-    {% if member_count %}
-      <div class="usa-card__members">
-        <span class="usa-card__member-count">
-          <svg class="usa-card__members-icon usa-icon dg-icon dg-icon--large margin-bottom-05" aria-hidden="true" focusable="false">
-            <use xlink:href="/{{ active_theme_path() }}/static/uswds/img/sprite.svg#groups"></use>
-          </svg>
-          {{ member_count }}
-          {{ 'Members'|t }}
-        </span>
-      </div>
-    {% endif %}
-    {% if summary %}
-      <div class="usa-card__body">
-        <p>{{ summary }}</p>
-      </div>
-    {% endif %}
-    <div
-      class="usa-card__footer">
-      <a href="{{ url }}" class="usa-button usa-button--outline">
-        {{ 'Connect with'|t }}
-        {{ title }}
-      </a>
-    </div>
-  </div>
-</div>
-{# {% elseif highlight == "Off" and page_weight == '1' %}
-  <ul class="communities-of-practice--list usa-collection">
-    <li class="usa-collection__item">
-      <div class="usa-collection__body">
-        <h2 class="usa-collection__heading">
-          <a class="usa-link usa-link--external" href="{{ redirect }}" title="{{ title }}">
-            {{ title }}
-          </a>
-        </h2>
-        <p class="usa-collection__description">
-          {{ summary }}
-        </p>
-      </div>
-    </li>
-  </ul>
-{% endif %} #}
+{# Caching the data in content object #}
+{% set savedCache = content|render %}
+
+{{ include('@digital_gov/partials/featured-communities-card.html.twig', {
+  heading: content.field_dg_shortname|field_value|default(label|field_value),
+  logo: content.field_dg_logo|field_value,
+  member_count: content.field_members|field_value,
+  summary: content.field_summary|field_value,
+  acronym: content.field_dg_acronym|field_value,
```

Comment:
Noticed two things compared to digital.gov
1. member count needs to have a comma in thousands place - number_format() or twig equivalent
2. there's a `<b>` ta after "Connect with" and the community name.

![image](https://github.com/user-attachments/assets/84fa2489-0de6-4266-9331-66bcb1711b15)

Reply:
also,not sure I added the comment on the right template

Reply:
finally, should we re-use teaser for a card display in one place. Can you ahve two card display mode variants like we have for guides?

Reply:
finally finaly - I didn't see an SVG image on the card in `/communities`. Can you export the default content with an image?

Reply:
Request revisions are resolved. 

I confirmed the icon is included in the codebase already
Node
https://github.com/GSA/digital-gov-drupal/blob/feature/DIGITAL-246-homepage-communities/web/modules/custom/default_content_config/content/node/623a2982-602b-4762-a59e-9a7c30c64f36.yml

File
https://github.com/GSA/digital-gov-drupal/blob/feature/DIGITAL-246-homepage-communities/web/modules/custom/default_content_config/content/file/3f447040-a671-4acf-85c1-c2301f9eb7b3.yml
and
https://github.com/GSA/digital-gov-drupal/blob/feature/DIGITAL-246-homepage-communities/web/modules/custom/default_content_config/content/file/communities-web-managers.svg

Let's chat more about the reuse of teaser for card display ideas.